index.rst

```diff
@@ -632,15 +641,76 @@ The disassembler function signature is:
             The object that will be disassembled into component parts.
         dataId : dict
             The dataId that is being used to persist this object.
-        componentDict : dict
-            A dict to populate with the components that should be persisted for
-            this composite.
+        componentInfo : dict of componentDatasetName to ComponentInfo
+            A dict of ComponentInfo instances to populate with the components
+            that should be persisted for this composite. Keys are the component
+            names as defined by the datasetType in the policy. Values are
+            ComponentInfo objects.
```

Comment:
Should mention that only `ComponentInfo.obj` is to be modified; the rest must be left alone.